romanesco/format/netcdf/validate_binary.json

```diff
@@ -0,0 +1,6 @@
+{
+    "inputs": [{"name": "input", "type": "netcdf", "format": "binary"}],
+    "outputs": [{"name": "output", "type": "boolean", "format": "boolean"}],
```

Comment:
Everything needs a format, even in trivial cases like this. I've defaulted to using the same name as the type for the "natural" type. "inline" may also be a decent way to describe it.

For the "boolean" type (similar for "number") there are two formats: "boolean" and "json". The difference is:

```json
```